src/thread_task.rs

```diff
@@ -528,26 +525,20 @@ unsafe fn check_tile(t_idx: Rav1dTaskIndex, f: &Rav1dFrameData, frame_mt: c_int)
                 };
                 let max = cmp::max(y, uv);
                 if max == i32::MIN {
-                    current_block_14 = 7651349459974463963;
-                } else {
-                    lowest = iclip(max, 1 as c_int, f.refp[n as usize].p.p.h) as c_uint;
-                    current_block_14 = 2370887241019905314;
+                    tasks[t_idx].deps_skip += 1;
```

Comment:
Can you just set `tasks[t_idx].deps_skip = 7` at the end of the loop?

Reply:
The loop can exit early so I don't think that would be right. Even if it was, it wouldn't make a difference.